src/briefcase/integrations/android_sdk.py

```diff
@@ -1265,11 +1254,11 @@ def logcat(self):
                     "-s",
                     self.device,
                     "logcat",
-                    "-s",
-                    "MainActivity:*",
-                    "stdio:*",
-                    "Python:*",
-                ],
+                    "--pid",  # This option is available since API level 24.
+                    pid,
+                ]
+                # Filter out some noisy and useless tags.
+                + [f"{tag}:S" for tag in ["EGL_emulation"]],
```

Comment:
`EGL_emulation` is a tag which generates a message [like this](https://stackoverflow.com/questions/70376084) about once per second on the API level 31 emulator.

Reply:
Interesting... I'm not seeing that on my Android 31 emulator or my Android 8.1 test device. The internet seems to suggest it's a common piece of line noise, though, so we might as well filter it.

Reply:
I guess it depends on what graphics acceleration mode the emulator is using.